app/src/main/java/com/babylon/wallet/android/presentation/wallet/WalletViewModel.kt

```diff
@@ -110,7 +105,7 @@ class WalletViewModel @Inject constructor(
             }
         }
         observePrompts()
-        observeAccounts()
+        observeWallet()
```

Comment:
my only objection is here. "wallet" is quite generic. It seems like observing everything in the wallet?
If not `observeAccounts` then maybe `observeWalletAssets`?